src/components/editor/GroupChoiceModal.jsx

```diff
@@ -34,16 +34,14 @@ const GroupChoiceModal = (props) => {
     event.preventDefault()
   }
 
-  const classes = ['modal', 'fade']
   let display = 'none'
```

Comment:
Same as above and elsewhere.

Reply:
And perhaps this should be made a function in ModalCss as well?

Reply:
Done, using new `useDisplayStyle` hook